tests/acceptance/features/bootstrap/SharingNgContext.php

```diff
@@ -329,33 +329,36 @@ public function userSetsOrUpdatesFollowingPasswordForLastLinkShareUsingTheGraphA
 	}
 
 	/**
-	 * @When /^user "([^"]*)" removes the share permission of (user|group) "([^"]*)" from (file|folder) "([^"]*)" of space "([^"]*)" using the Graph API$/
-	 *
 	 * @param string $sharer
 	 * @param string $shareType (user|group)
-	 * @param string $sharee can be both user or group
 	 * @param string $resourceType
 	 * @param string $resource
 	 * @param string $space
+	 * @param string|null $sharee can be both user or group
 	 *
-	 * @return void
+	 * @return ResponseInterface
+	 * @throws GuzzleException
 	 * @throws JsonException
-	 * @throws \GuzzleHttp\Exception\GuzzleException
 	 */
-	public function userRemovesSharePermissionOfUserFromResourceOfSpaceUsingGraphAPI(
+	public function removeSharePermissionOfAResource(
 		string $sharer,
 		string $shareType,
-		string $sharee,
 		string $resourceType,
 		string $resource,
-		string $space
-	): void {
+		string $space,
+		?string $sharee = null
+	): ResponseInterface {
 		$spaceId = ($this->spacesContext->getSpaceByName($sharer, $space))["id"];
 		$itemId = ($resourceType === 'folder')
 			? $this->spacesContext->getResourceId($sharer, $space, $resource)
 			: $this->spacesContext->getFileId($sharer, $space, $resource);
-		$permId = $this->featureContext->shareNgGetLastCreatedUserGroupShareID();
-		$this->featureContext->setResponse(
+		$permId = null;
+		if ($shareType === 'link') {
+			$permId = $this->featureContext->shareNgGetLastCreatedLinkShareID();
+		} else {
+			$permId = $this->featureContext->shareNgGetLastCreatedUserGroupShareID();
+		}
```

Comment:
```suggestion
		$permId = ($shareType === 'link') ? $this->featureContext->shareNgGetLastCreatedLinkShareID() : $this->featureContext->shareNgGetLastCreatedUserGroupShareID();
		}
```
Or we can do in this way